radio/src/targets/common/arm/stm32/bootloader/CMakeLists.txt

```diff
@@ -88,11 +95,11 @@ if(PCB STREQUAL X10 OR PCB STREQUAL X12S OR PCB STREQUAL NV14)
       ../../../../../targets/${TARGET_DIR}/extmodule_helper.cpp
       )
   endif()
-  
+
   if(DEBUG)
     set(BOOTLOADER_SRC
       ${BOOTLOADER_SRC}
-      ../../../../../${STM32LIB_DIR}/STM32F4xx_StdPeriph_Driver/src/stm32f4xx_usart.c
+#      ../../../../../${STM32LIB_DIR}/STM32F4xx_StdPeriph_Driver/src/stm32f4xx_usart.c
```

Comment:
This seems adding this source twice. Causing issues when compiling with -DDEBUG=y